bin/jest.js

```diff
@@ -134,7 +134,7 @@ var cwd = process.cwd();
 // Is the cwd somewhere within an npm package?
 var cwdPackageRoot = cwd;
 while (!fs.existsSync(path.join(cwdPackageRoot, 'package.json'))) {
-  if (cwdPackageRoot === '/') {
+  if (cwdPackageRoot === '/' || cwdPackageRoot.match(/[A-Z]:\\/)) {
```

Comment:
Should the regexp contain `^`?

`/^[A-Z]:\\/`

(also, worth adding a small comment that this is for Windows; probably we should have set of helper methods based on `os.platform`, but for now this in place special case for Windows is OK)